### ISSUE TITLE
JSDOC: Add documentation generation to package.info

### ISSUE THREAD

**Issue by rmhrisk** [PeculiarVentures/node-webcrypto-ossl#67]
We can add generation of JSDOC derived documentation via docdash: https://github.com/clenemt/docdash

"script": {
  "generate-docs": "node_modules/.bin/jsdoc -c jsdoc.json"
}
This way we can have standalone documentation showing how to use the library.

**Comment by hildjj** [comment]
Presumably, you want to use [typedoc](https://github.com/TypeStrong/typedoc) or [tsdoc](https://github.com/xperiments/TSDoc), not jsdoc?  I don't know if the docdash theme will work with the output of either of those.

**Comment by rmhrisk** [comment]
I was a bit over zealous with copy and paste, the goal is to have generated documents.

If the Docdash theme will not work we have some guys we can make one that is visually similar that will work with typedoc/tsdoc.

Do you have experience with either?

**Comment by hildjj** [comment]
No.  They both look... early.

**Comment by rmhrisk** [comment]
Agreed, was hoping you might be able to provide assurance it was OK :)
